protocol-definitions/Jet.yaml

```diff
@@ -301,3 +301,23 @@ methods:
           nullable: false
           since: 2.5
           doc: ''
+  - id: 16
+    name: isJobUserCancelled
+    since: 2.6
+    doc: ''
```

Comment:
Minor: I see that we haven't added any documentation for any of the jet codecs, but it might be good to document this protocol definition and its request and response parameters for future reference

Reply:
that is true. I think it would be better to do it in a separate pull request.